flows/flows.go

```diff
@@ -43,8 +43,8 @@ type Flow struct {
 	output    int
 	resubmit  int
 	drop      bool
-
-	raw string
+	local     bool
```

Comment:
Keep the space between `local` and `raw` please

Reply:
looks like I forgot to format this one.